cli/cmd/generate.go

```diff
@@ -355,7 +355,7 @@ func injectInitializer(resources []any) error {
 	for _, resource := range resources {
 		switch r := resource.(type) {
 		case *applyappsv1.StatefulSetApplyConfiguration:
-			if r.Spec != nil && r.Spec.Template != nil &&
+			if r.Spec != nil && r.Spec.Template != nil && r.Spec.Template.Annotations != nil &&
```

Comment:
Should we add a test for this to make sure that we don't regress? I'm fine with doing this in a follow up PR if we want to still release today.